src/crate/theme/rtd/crate/sidebartoc.html

```diff
@@ -16,14 +16,19 @@
     </div>
   </li>
 
-  <!-- Home. -->
-  {% if project == 'CrateDB: Guide' and pagename == 'home/index' %}
-    <li class="navleft-item current"><a class="current-active" href="/docs/guide/home/">Overview</a></li>
+  <!-- Section A: -->
+  {% if project == 'CrateDB: Guide' %}
+    <!-- Show all TOC entries without the Guide heading -->
+    <li class="current toctree-l0">
+      {{ toctree(maxdepth=-1|toint, titles_only=True, collapse=True, includehidden=theme_globaltoc_includehidden|tobool) }}
+    </li>
   {% else %}
+    <!-- Show Guide sections as links in other projects -->
     <li class="navleft-item"><a href="/docs/guide/home/">Overview</a></li>
+    <li class="navleft-item"><a href="/docs/guide/start/">Getting Started</a></li>
+    <li class="navleft-item"><a href="/docs/guide/guides/">Guides</a></li>
```

Comment:
I am not particular happy with "naming things" about "Guides". I am considering it a positively tainted name per Diátaxis now, so let's find a different one.

- GH-635

Reply:
Maybe within this patch, the relocation of `docs/index.md` to `docs/guides/index.md` can be handled differently? I know this solution is hacky as a whole, but maybe we can minimize its impact on this particular detail?

- https://github.com/crate/cratedb-guide/pull/348

Reply:
Sure we can update the naming to support other new PRs already.

I'm not sure what the alternative would be for relocating the guides index? It seems a natural to me. We could even relocate the guides content to that folder if we want the physical structure to closer represent the TOC. But going forward I envision we will relocate many of the pages under Guides (or whatever it should be called)